### PR TITLE
use identity file for ssh connection

### PR DESCRIPTION
**Pull request by arnaud-deprez**
Hi,

I've used your hawesome docker-compose to play with kubernetes on my local machine. 
I decided to share with you my little change.
I updated the way it connects to the boot2docker instance by using the machine identity file.
So we don't need to type the ssh password anymore.

Rgds,

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Created by .ignore support plugin (hsz.mobi)
```
Comment (vyshane) [inline .gitignore]: Can you remove this file?

File: scripts/docker-machine-port-forwarding.sh [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 function forward_port_if_not_forwarded {
     port=$1
-    forward_port_command="ssh -f -N -L $port:localhost:$port docker@$(docker-machine ip $(docker-machine active))"
+    machine=$(docker-machine active)
+    forward_port_command="ssh -f -N -i ~/.docker/machine/machines/$machine/id_rsa -L $port:localhost:$port docker@$(docker-machine ip $machine)"
```
Comment (vyshane) [inline scripts/docker-machine-port-forwarding.sh]: Using `~` in the path breaks the conditional that checks for an existing port forward. Can you change it to use `$HOME` instead? I.e:

``` bash
forward_port_command="ssh -f -N -i $HOME/.docker/machine/machines/$machine/id_rsa -L $port:localhost:$port docker@$(docker-machine ip $machine)"
```

**Comment by vyshane** [comment]
Thanks for the PR :) Good idea, using the Machine's identity file. Makes the whole thing more convenient.

I had a couple of comments.

**Comment by arnaud-deprez** [comment]
No problem for your comments, here's the new commit.

**Pull request merged**